src/main/java/org/spburegistry/backend/entity/BaseEntity.java

```diff
@@ -0,0 +1,29 @@
+package org.spburegistry.backend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+@Data
+@MappedSuperclass
+@AllArgsConstructor
+@NoArgsConstructor
+public abstract class BaseEntity implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @UpdateTimestamp
+    @Column(name = "last_update_time", nullable = false)
+    private Timestamp lastUpdateTime;
```

Comment:
I think that better use Date type of value. And do like this may be https://stackoverflow.com/a/39427923/20352174

Reply:
Done.